uni/lib/controller/parsers/parser_library_occupation.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:uni/model/entities/library_occupation.dart';
+
+Future<LibraryOccupation> parseLibraryOccupationFromSheets(
+    Response response) async {
+  final js = response.body.split('\n')[1];
+  const skip = 'google.visualization.Query.setResponse(';
+  const numFloors = 6;
+  const List<int> max = [69, 105, 105, 105, 40, 64];
```

Comment:
Can you extract this values from the sheets?

Reply:
I don't know why, but using the `gviz` link those values do not available.

Reply:
Can you double check this @jlcrodrigues?

Reply:
Found a different way to query the sheets so this is no longer a problem.